forge-gui/res/cardsfolder/c/caged_sun.txt

```diff
@@ -4,6 +4,6 @@ Types:Artifact
 K:ETBReplacement:Other:ChooseColor
 SVar:ChooseColor:DB$ ChooseColor | Defined$ You | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a color.
 S:Mode$ Continuous | Affected$ Creature.ChosenColor+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Creatures you control of the chosen color get +1/+1.
-T:Mode$ TapsForMana | ValidCard$ Land | Produced$ ChosenColor | NoTapCheck$ True | Player$ You | Execute$ TrigMana | TriggerZones$ Battlefield | Static$ True | TriggerDescription$ Whenever a land's ability causes you to add one or more mana of the chosen color, add one additional mana of that color.
-SVar:TrigMana:DB$ Mana | Produced$ Chosen | Amount$ 1 | Defined$ You
+T:Mode$ ManaAdded | ValidSource$ Land | Produced$ ChosenColor | Player$ You | Execute$ TrigMana | TriggerZones$ Battlefield | Static$ True | TriggerDescription$ Whenever a land's ability causes you to add one or more mana of the chosen color, add one additional mana of that color.
```

Comment:
New ruling:
>An ability that triggers from the resolution of an activated mana ability is also a mana ability. This means that if the land's ability that caused you to add mana was an activated mana ability, Caged Sun's triggered ability won't use the stack and can't be responded to. 

So it can be either Static or not...

I think we could just add a second trigger and then do `ValidSA$ SpellAbility.nonManaAbility`?